lib/widgets/input_field.dart

```diff
@@ -51,28 +47,11 @@ class InputField extends StatelessWidget {
         },
         textInputAction: textInputAction,
         validator: (value) {
-          if (value == null || value.isEmpty) {
-            return 'Please enter some text';
+          if (value?.isEmpty ?? true) {
+            value = value?.trim();
+            return AppStrings.emptyError;
```

Comment:
Please check the application of trim to be appropriate.

Reply:
This can be accommodate in the below function itself, please see.

Reply:
resolvedr